### PR TITLE
Improvements on animations.

### LINKED ISSUE #29: better animations
I think the client should use a better method for animating cell position changes.

### PR DESCRIPTION
**Pull request by henopied**
Closes #29.

### CODE REVIEW

File: client/js/ball-view.js [hunk 1]
```diff
@@ -36,11 +36,11 @@ class BallView {
...
     this.ball.on('disappear', () => this.disappear());
     this.ball.on('move', (oldX, oldY, newX, newY) => {
-      this.x.set(newX, 100);
-      this.y.set(newY, 100);
+      this.x.set(newX, 120);
+      this.y.set(newY, 120);
     });
     this.ball.on('resize', (oldSize, newSize) => {
-      this.s.set(newSize, 100);
+      this.s.set(newSize, 200);
```
Comment (heroandtn3) [inline client/js/ball-view.js]: 200?

Comment (henopied) [inline client/js/ball-view.js]: Yes, it seems like a much more pleasant rate of size change but if you wish I could change it.

Comment (heroandtn3) [inline client/js/ball-view.js]: Yes, vanilla agar version uses 120 for size too, so I think this should be 120.

Comment (henopied) [inline client/js/ball-view.js]: They also changed the easing though in more resent versions (not the one I linked in the issue.) I am wondering if we can re-create that easing method.

Comment (heroandtn3) [inline client/js/ball-view.js]: Easing? Sorry I do not get your point. Can you refer to original code from agar.io?

Comment (henopied) [inline client/js/ball-view.js]: [this should help a bit](https://gist.github.com/Apostolique/1edcb261470b7a73c4b4#file-gistfile1-txt-L1495)

Comment (heroandtn3) [inline client/js/ball-view.js]: Thanks :+1:

**Comment by heroandtn3** [comment]
Thank you! I'm going to merge this.

**Pull request merged**